### ISSUE TITLE
[🐛BUG] PLM inconsistency with paper

### ISSUE THREAD

**Issue by jackmedda** [tail-unica/hopwise#238]
The paper proposing PLM (*Path Language Modeling over Knowledge Graphs for Explainable Recommendation*) includes 3 steps that are not currenly evisioned by our implementation.

### Path augmentation
> Although the original training paths include enough historical records regarding users and items, they are insufficiently diverse to explore new possibilities within the KG. To further improve the robustness of the trained PLM-Rec model, it is necessary to augment the input sequences. Hence, we perform sequence augmentation by randomly substituting a style or feature entity with a similar entity. We use Sentence-BERT \[36] as a metric to measure the semantic similarity and set a similarity threshold θ to decide augmentation candidates for all style and feature entities. The union of original and augmented path sequences serves as the final overall training data.

### Nucleus sampling
Section 5.3 of the paper

### Pretrained Entity embeddings
The entity embedding size d is set to 100, and the entity embeddings are initialized with pretrained KG embeddings [2].  ([2] is TransE)
